llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1918,6 +1928,18 @@ class VPWidenIntOrFpInductionRecipe : public VPWidenInductionRecipe {
   VPValue *getVFValue() { return getOperand(2); }
   const VPValue *getVFValue() const { return getOperand(2); }
 
+  // TODO: Remove once VPWidenIntOrFpInduction is fully expanded in
+  // convertToConcreteRecipes.
+  VPValue *getStepVector() { return getOperand(3); }
```

Comment:
At the moment, it always must be a `VPInstructionWithType`, right? Would changing the return type to `VPInstructionWithType` remove some casts at use sites?

Also asser that it is a `StepVector` opcode here, rather than at use sites?

Reply:
Done, thanks